fiftyone/migrations/revisions/v1_4_0.py

```
@@ -0,0 +1,49 @@
+                f"{source_field}": {
+                    "$regex": f"\\{LOG_FILE_SUFFIX}$",
+                    "$exists": True,
+                },
+            },
+            {"$rename": {f"{source_field}": f"{target_field}"}},
+        )
+    except Exception as e:
+        raise RuntimeError(
+            f"Failed to replace '{target_field}' with '{source_field}' "
+            f"for dataset '{dataset_name}'. Reason: {e}"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)